climada/hazard/centroids/centr.py

```diff
@@ -820,386 +638,274 @@ def get_closest_point(self, x_lon, y_lat, scheduler=None):
         idx_close : int
             Index of centroid in internal ordering of centroids.
         """
-        if self.meta:
-            if not self.lat.size or not self.lon.size:
-                self.set_meta_to_lat_lon()
-            i_lat, i_lon = rasterio.transform.rowcol(self.meta['transform'], x_lon, y_lat)
-            i_lat = np.clip(i_lat, 0, self.meta['height'] - 1)
-            i_lon = np.clip(i_lon, 0, self.meta['width'] - 1)
-            close_idx = int(i_lat * self.meta['width'] + i_lon)
-        else:
-            self.set_geometry_points(scheduler)
-            close_idx = self.geometry.distance(Point(x_lon, y_lat)).values.argmin()
+        close_idx = self.geometry.distance(Point(x_lon, y_lat)).values.argmin()
         return self.lon[close_idx], self.lat[close_idx], close_idx
 
-    def set_region_id(self, scheduler=None):
-        """Set region_id as country ISO numeric code attribute for every pixel or point.
+    # NOT REALLY AN ELEVATION FUNCTION, JUST READ RASTER
+    def get_elevation(self, topo_path):
+        """Return elevation attribute for every pixel or point in meters.
```

Comment:
Maybe we can just remove it? It doesn't seem to be used anywhere, and it's really extremely simplistic, just calling `u_coord.read_raster_sample` with trivial arguments.